custom_components/tuya_local/devices/pilot_rp5_bath_thermostat_min.yaml

```diff
@@ -0,0 +1,82 @@
+name: Pilot RP5 Thermostat (minimal+presets)
+products:
+  - id: c03zek9b5daz7omr
+    name: Pilot Thermostat RP5
+
+entities:
+  - entity: climate
+    name: Bathroom Thermostat
+    icon: "mdi:radiator"
+    dps:
+      # --- HVAC mode (off / heating / auto) ---
+      - id: 101
+        type: string
+        name: hvac_mode
+        mapping:
+          - dps_val: "Standby"
+            value: "off"
+          - dps_val: "Programming"
+            value: "auto"
+          # All heating modes -> heat
+          - dps_val: "Comfort"
+            value: "heat"
+          - dps_val: "Comfort_1"
+            value: "heat"
```

Comment:
To ensure the behaviour is not random, all but one of the mappings to `heat` should be marked with `hidden: true`.
This could then be abbreviated by using a default mapping:
```
mapping:
  - dps_val: Standby
    value: "off"
  - dps_val: Programming
    value: auto
  - dps_val: Comfort
    value: heat
  - value: heat
    hidden: true
```